CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**1.2.6 - 09/05/23**
+
+ - Update to run linting before tests for builds.
+
```

Comment:
There's no release associated with this, since it's just a github workflow change, so no need to add it to the changelog (or release).